sky/utils/command_runner.py

```diff
@@ -207,6 +212,92 @@ def _get_command_to_run(
         command_str = ' '.join(command)
         return command_str
 
+    def _rsync(
+            self,
+            source: str,
+            target: str,
+            node_destination: str,
+            up: bool,
+            rsh_option: str,
+            # Advanced options.
+            log_path: str = os.devnull,
+            stream_logs: bool = True,
+            max_retry: int = 1,
+            prefix_command: Optional[str] = None,
+            get_remote_home_dir: Callable[[], str] = lambda: '~'):
```

Comment:
```suggestion
            get_remote_home_dir: Callable[[], str] = lambda: '~') -> None:
```
nit; to enable mypy check